jujugui/static/gui/src/app/components/header-help/header-help.js

```diff
@@ -58,7 +58,7 @@ class HeaderHelp extends React.Component {
   _generateIssuesLink() {
     let label = 'File Issue';
     let link = 'https://github.com/juju/juju-gui/issues';
-    if (this.props.gisf && this.props.user) {
+    if (this.props.user) {
```

Comment:
I am not sure about pointing gijoe users to the JAAS support page. I'd like them not to assume that we provide support for their own controllers.